tests/test_create.py

```diff
@@ -29,7 +36,8 @@ def test_create_environment_from_lockfile_pixi_metadata(tmp_path: Path) -> None:
     env_record_path = tmp_path / "conda-meta" / "tzdata-2025b-h78e105d_0.json"
     assert env_record_path.is_file()
     data = json.loads(env_record_path.read_bytes())
-    assert data["license"] == "ONLY_IN_LOCKFILE"
+    # lockfile overrides license but we ignore this
+    assert data["license"] == "LicenseRef-Public-Domain"
```

Comment:
This is incorrect. The lockfile is the source of truth for package information. See discussion in #4

Reply:
Any of these fields can be patched by repodata hotfixing. The most reliable source of information about the packages is the lockfile. A lockfile should be created from the most up-to-date information about the packages included, e.g. hotfixed repodata.

The other sources of package information are less reliable:
* The package itself : does not reflect any changed made from repodata hotfixing
* The package cache : Reflects the information about the package that was known at the time the package was first downloaded. This could be from repodata or from the package itself. It is is from repodata it is possible that it has been patched after the package entered the cache in which case the cached data is incorrect.